src/blockchain/payments.js

```diff
@@ -79,9 +84,10 @@ const payments = app => ({
       throw new Error(`No donation found with reference: ${reference}`);
     }
 
-    const { ownerTypeId: milestoneId } = donation;
+    const { ownerTypeId: milestoneId, _id: donationId } = donation;
 
-    const { campaignId } = await milestoneModel.findById(milestoneId, ['campaignId']);
+    const milestone = await milestoneModel.findById(milestoneId);
```

Comment:
This was wrong?
```
const { campaignId } = await milestoneModel.findById(milestoneId, ['campaignId']);
```

Reply:
No it's ok, 
it was my mistake, I put it back